var/spack/repos/builtin/packages/eospac/package.py

```diff
@@ -42,4 +42,18 @@ def url_for_version(self, version):
 
     def install(self, spec, prefix):
         with working_dir('Source'):
-            make('prefix=%s' % self.spec.prefix, 'install')
+            targets = ['install']
+            targets.append('CC={0}'.format(self.compiler.cc))
+            targets.append('CXX={0}'.format(self.compiler.cxx))
+            targets.append('F77={0}'.format(self.compiler.f77))
+            targets.append('F90={0}'.format(self.compiler.fc))
+            targets.append('prefix={0}'.format(self.spec.prefix))
+            targets.append('INSTALLED_LIBRARY_DIR={0}'.format(
+                self.spec.prefix.lib))
+            targets.append('INSTALLED_INCLUDE_DIR={0}'.format(
+                self.spec.prefix.include))
+            targets.append('INSTALLED_EXAMPLE_DIR={0}'.format(
+                self.spec.prefix.example))
+            targets.append('INSTALLED_BIN_DIR={0}'.format(
+                self.spec.prefix.bin))
```

Comment:
You could replace all of these appends with a much simpler:
```python
targets = [
    'install',
    'CC={0}'.format(spack_cc),
    'CXX={0}'.format(spack_cxx),
    ....
]
```
or even pass these directly to `make`.

Reply:
Done